README.md

```diff
@@ -1,11 +1,11 @@
-# Healer data service (0.0.1)
+# Healer data processor (0.0.2)
```

Comment:
I cannot select the following block for comment
```yaml
      type: NodePort
      endpoints:
        - name: 'grpc'
          targetPort: 8080
          nodePort: <free port>
```

but we should change the `type` to ClusterIP and remove the `nodePort` parameter